problem2.py

```diff
@@ -1 +1,39 @@
+"""
+-------------------------------------------------------------------------------
+Name:   problem2.py
+Purpose:  Computes wha
 
+Author: Khatchadourian.N
+
+Created:  23/02/2021
+------------------------------------------------------------------------------
+"""
+print("******* Triangle Detector ********")
+
+#Gets the side lengths from the user
+first_side = int(input("Enter the length of the first side: "))
+second_side = int(input("Enter the length of the second side: "))
+third_side = int(input("Enter the length of the third side: "))
+
+#Checks which side is the biggest then checks if two sides add up to be greater than the side that is the biggest
+if third_side > first_side and second_side:
```

Comment:
logic for check that one side is greater than the other two sides is incorrect.  you need to be explicit:
```
if third_side > first_side and third_side >second_side:
```